src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-*",
```

Comment:
@CIPop What APIs are changing exactly? 
We are using Socket, NetworkStream, TcpClient from the Sytem.Net.Sockets

They are referenced in 
System\Data\SqlClient\SNI\SNITcpHandle.cs

Are there going to be any changes there?